### PR TITLE
Removed expired links and added example apps

### PR DESCRIPTION
**Pull request by usama9500**
- Added two examples (Web App and Website)
- Removed old SDK references

### CODE REVIEW

File: lib/js-sdk/README.md [hunk 1]
```diff
@@ -69,12 +69,17 @@ Add the zus-sdk package for use in your project
 ```
 yarn add @zerochain/zus-sdk
 ```
+Here's the updated **SDK Reference** section with the mentioned addition:
```
Comment (ashuvssut) [inline lib/js-sdk/README.md:72]:
```suggestion
```

File: lib/js-sdk/README.md [hunk 2]
```diff
@@ -69,12 +69,17 @@ Add the zus-sdk package for use in your project
...
+
+---
 
 ## SDK Reference
 
 Zus JS-sdk provides many functions for implemention blobber,allocation and file operation functionalities in web applications.
```
Comment (ashuvssut) [inline lib/js-sdk/README.md:78]:
```suggestion
Zus JS-sdk provides many functions for implementing blobber, allocation and file operation functionalities in web applications.
```

File: lib/js-sdk/README.md [hunk 3]
```diff
@@ -69,12 +69,17 @@ Add the zus-sdk package for use in your project
...
 
-- See [gitbook](https://docs.zus.network/guides/zus-js-sdk/sdk-reference) for extensive list.
+To get a comprehensive understanding of the functions exposed by the JS SDK, you can refer to the [index file](https://github.com/0chain/zus-js-sdk/blob/main/lib/js-sdk/src/index.ts) in the SDK repository.
```
Comment (ashuvssut) [inline lib/js-sdk/README.md:80]:
```suggestion
For a detailed understanding of the functions provided by the JS SDK, please refer to the JS Doc comments in the [index.ts file](https://github.com/0chain/zus-js-sdk/blob/main/lib/js-sdk/src/index.ts)
```

File: lib/js-sdk/README.md [hunk 4]
```diff
@@ -69,12 +69,17 @@ Add the zus-sdk package for use in your project
...
+---
 
 ## SDK Reference
 
 Zus JS-sdk provides many functions for implemention blobber,allocation and file operation functionalities in web applications.
 
-- See [gitbook](https://docs.zus.network/guides/zus-js-sdk/sdk-reference) for extensive list.
+To get a comprehensive understanding of the functions exposed by the JS SDK, you can refer to the [index file](https://github.com/0chain/zus-js-sdk/blob/main/lib/js-sdk/src/index.ts) in the SDK repository. 
+
+Most of the functions are well-documented using JS Docs comments, making it easy to understand their usage and parameters.
```
Comment (ashuvssut) [inline lib/js-sdk/README.md:82]:
```suggestion
Most of the exposed SDK functions are well-documented with JS Doc comments, detailing their usage, parameter types, and return types.
```

**Review by ashuvssut — CHANGES_REQUESTED** [review]
are there any gitbook changes?

**Comment by usama9500** [comment]
> are there any gitbook changes?

No changes on gitbook. This PR is for readme file.  For gitbook, I will add new pages for SDK reference directly.

**Pull request merged**